tb/tb_avalon_enforcer.sv.bak

```diff
@@ -0,0 +1,90 @@
+//////////////////////////////////////////////////////////////////
```

Comment:
you should delete this (A possible hurdle: .gitignore might make you trouble removing it as it might "ignore" this file existence. if this happens you should edit the .gitignore locally, remove this file from your repo, commit only the changes in this file and then push)